frontend/src/utils/formValidation.ts

```diff
@@ -30,49 +30,31 @@ export const FORM_TITLE_VALIDATION_RULES: UseControllerProps['rules'] = {
   },
 }
 
-export const createAdminEmailValidationTransform = () => {
-  const transform = {
-    // Combine and display all emails in a single string in the input field.
-    input: (value: string[]) => value.join(','),
-    // Convert joined email string into an array of emails.
-    output: (value: string) =>
-      value
-        .replace(/\s/g, '')
-        .split(',')
-        .map((v) => v.trim()),
-  }
-
-  const rules: UseControllerProps['rules'] = {
-    validate: {
-      required: (emails: string[]) => {
-        return (
-          emails.filter(Boolean).length > 0 ||
-          'You must at least enter one email to receive responses'
-        )
-      },
-      valid: (emails: string[]) => {
-        return (
-          emails.filter(Boolean).every((e) => validator.isEmail(e)) ||
-          'Please enter valid email(s) (e.g. me@example.com) separated by commas.'
-        )
-      },
-      duplicate: (emails: string[]) => {
-        return (
-          new Set(emails).size === emails.length ||
-          'Please remove duplicate emails.'
-        )
-      },
-      maxLength: (emails: string[]) => {
-        return (
-          emails.length <= MAX_EMAIL_LENGTH ||
-          'Please limit number of emails to 30.'
-        )
-      },
+export const ADMIN_EMAIL_VALIDATION_RULES: UseControllerProps['rules'] = {
+  validate: {
+    required: (emails: string[]) => {
+      return (
+        emails.filter(Boolean).length > 0 ||
+        'You must at least enter one email to receive responses'
+      )
     },
-  }
-
-  return {
-    rules,
-    transform,
-  }
+    valid: (emails: string[]) => {
+      return (
+        emails.filter(Boolean).every((e) => validator.isEmail(e)) ||
+        'Please enter valid email(s) (e.g. me@example.com) separated by commas.'
+      )
+    },
+    duplicate: (emails: string[]) => {
+      return (
+        new Set(emails).size === emails.length ||
+        'Please remove duplicate emails.'
+      )
+    },
+    maxLength: (emails: string[]) => {
+      return (
+        emails.length <= MAX_EMAIL_LENGTH ||
+        'Please limit number of emails to 30.'
+      )
+    },
```

Comment:
I hadn't spotted this before, but since the 2 validators above filter the array for truthy entries, should we not do the same filtering here too too, to be consistent on what we accept?

```suggestion
    duplicate: (emails: string[]) => {
      const truthyEmails = emails.filter(Boolean)
      return (
        new Set(truthyEmails).size === truthyEmails.length ||
        'Please remove duplicate emails.'
      )
    },
    maxLength: (emails: string[]) => {
      return (
        emails.filter(Boolean).length <= MAX_EMAIL_LENGTH ||
        'Please limit number of emails to 30.'
      )
    },
```

Reply:
done ed08f8c